cosmos/mantra-dukong.json

```diff
@@ -0,0 +1,56 @@
+{
+    "chainId": "mantra-dukong-1",
+    "chainName": "MANTRA Dukong Testnet",
+    "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/mantra/chain.png",
```

Comment:
You need an image for each chain identifier
For `mantra-dukong`, please also put the image in `images/mantra-dukong` and change the url to ‘https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/mantra-dukong/chain.png’

Reply:
thanks @blacktoast, that is done.